openshift4.yml

```diff
@@ -0,0 +1,12 @@
+classes:
+  - .common
+
+applications:
+  - prometheus
+
```

Comment:
This is not necessary, since we already add application `prometheus`in `common.yml`.